package.json

```diff
@@ -1,28 +1,32 @@
 {
   "name": "console-plugin-template",
   "version": "0.0.1",
+  "description": "Template project for OpenShift Console dynamic plugins",
   "private": true,
-  "repository": "git@github.com:spadgett/console-plugin-template.git",
   "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/openshift/console-plugin-template.git"
+  },
   "scripts": {
     "clean": "rm -rf dist",
-    "build": "yarn clean && NODE_ENV=production yarn ts-node node_modules/.bin/webpack",
-    "build-dev": "yarn clean && yarn ts-node node_modules/.bin/webpack",
-    "start": "yarn ts-node node_modules/.bin/webpack serve",
+    "build": "yarn clean && NODE_ENV=production yarn webpack",
+    "build-dev": "yarn clean && yarn webpack",
+    "start": "yarn webpack serve --progress",
     "start-console": "./start-console.sh",
     "i18n": "./i18n-scripts/build-i18n.sh && node ./i18n-scripts/set-english-defaults.js",
-    "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'",
-    "lint": "eslint ./src ./integration-tests --fix &&  stylelint \"src/**/*.css\" --allow-empty-input --fix",
-    "test-cypress": "cd integration-tests s && cypress open --env openshift=true",
+    "lint": "yarn eslint src integration-tests --fix && stylelint 'src/**/*.css' --allow-empty-input --fix",
+    "test-cypress": "cd integration-tests && cypress open --env openshift=true",
```

Comment:
Again not for this PR (but commenting since I see it): why do we need `openshift=true`? Isn't it always OpenShift?

@cyril-ui-developer Do you know?

Reply:
> Again not for this PR (but commenting since I see it): why do we need `openshift=true`? Isn't it always OpenShift?
> 
> @cyril-ui-developer Do you know?

@spadgett Not sure, but it is the same in [console](https://github.com/openshift/console/blob/master/frontend/package.json#L26)

Reply:
Right, in console, we had tests for both OpenShift and native k8s. It shouldn't be needed in this repo.